test/e2e/network/hostport.go

```diff
@@ -157,6 +157,18 @@ var _ = common.SIGDescribe("HostPort", func() {
 // create pod which using hostport on the specified node according to the nodeSelector
 // it starts an http server on the exposed port
 func createHostPortPodOnNode(f *framework.Framework, podName, ns, hostIP string, port int32, protocol v1.Protocol, nodeName string) {
+
+	var netexecArgs []string
+	var readinessProbePort int32
+
+	if protocol == v1.ProtocolTCP {
+		readinessProbePort = 8080
+		netexecArgs = []string{"--http-port=8080", "--udp-port=-1"}
+	} else {
+		readinessProbePort = 8008
+		netexecArgs = []string{"--http-port=8008", "--udp-port=8080"}
+	}
```

Comment:
we support SCTP too, the test only checks UDP though but do you mind fixing this function just for all protocols

Reply:
https://github.com/kubernetes/kubernetes/blob/1290d6a4c0c5cc4f758b428993d2c3760d9cf279/test/images/agnhost/netexec/netexec.go#L140-L141

Reply:
scratch this comment